src/plugins/irc-events/link.js

```diff
@@ -106,6 +106,21 @@ function parse(msg, preview, res, client) {
 
 		break;
 
+	case "audio/midi":
+	case "audio/mpeg":
+	case "audio/mpeg3":
+	case "audio/ogg":
+	case "audio/wav":
+	case "audio/x-mid":
+	case "audio/x-midi":
+	case "audio/x-mpeg":
+	case "audio/x-mpeg-3":
+		if (!`/^https:\/\//`.test(url)) {
```

Comment:
Why do you have string interpolation here?

This could be done without regex: `url.startsWith("https://")`